src/dbnode/storage/bootstrap/bootstrapper/fs/source.go

```diff
@@ -520,8 +526,6 @@ func (s *fileSystemSource) loadShardReadersDataIntoShardResult(
 			)
 			switch run {
 			case bootstrapDataRunType:
```

Comment:
Is it correct to fall through here? Seems like you'd be generating a `indexBlockSegment` for no reason? Although it looks like it was doing this previously, too

Reply:
I'll put a // pass commit in here.